internal/utils/net.go

```diff
@@ -1,12 +1,23 @@
 package utils
 
 import (
+	"bytes"
 	"errors"
 	"fmt"
 	"net"
+	"strconv"
+	"time"
 )
 
-func LocalIP() string {
+var(
+	LocalIP string
+)
+
+func init() {
+	LocalIP = getLocalIP()
+}
+
+func getLocalIP() string {
```

Comment:
move code of the func to init

Reply:
If we move the func to init, the inif func will mess up

Reply:
done